engine/src/main/java/org/terasology/world/chunks/Chunk.java

```diff
@@ -19,6 +19,9 @@
 import org.terasology.module.sandbox.API;
 
 /**
+ * Chunks are the smallest logical units of the worlds.
+ *
+ * Each chunk is 32x32x64 in dimensions, and is the default unit passed to world generator during world generation.
```

Comment:
and is the default unit passed to world generator during world generation. I'm not sure what this means exactly. I think you are talking about chunks as if they were a unit. The basic concept of a chunk is to allow the world to be partitioned into manageable blocks. maybe talk about a chunk in those terms rather than as a cubic unit.

Reply:
I think the size of a chunk is defined in `ChunkConstants`. Probably it is better to add a reference to ChunkConstants instead of duplicating the values in the documentation here.